src/librustdoc/clean/utils.rs

```diff
@@ -104,6 +104,7 @@ pub(crate) fn ty_args_to_args<'tcx>(
                     arg: index,
                 }),
             ))),
+            GenericArgKind::Const(ct) if let ty::ConstKind::Param(p) = ct.kind() && p.name == sym::host => None,
```

Comment:
Contrary to the HIR code path which filters out `#[rustc_host]` params, this filters out params named `host` which isn't quite correct, right? Could you leave a `FIXME(effects)` if it's too annoying to fix rn (requires checking the corresp. `ty::Generics` I think)?

Reply:
Since we're just looking for `ConstKind::Param`, we obviously don't hide host args like `false`, `true` or more complex const exprs. Could you add a `FIXME(effects)` for this as well / extend the previous one.

I only know of one case where this currently surfaces (without using `#![rustc_attrs]` + `#[rustc_host]`). Namely, documenting the cross-crate re-export of `pub fn h<T: Tr>() {}` where `Tr` is a const trait results in `pub fn h<T>() where T: Tr<true>` since we currently don't elide defaulted args in cross-crate scenarios (#80379). My PR #112463 would fix this specific case but it's blocked on performance.

(This would also surface with always-const bounds if they were introduced like `T: const Trait` → `T: Trait<false>`, etc.)